cvm/quantization/sym_pass.py

```diff
@@ -735,24 +735,79 @@ def _reduce_graph(sym, params, graph, inputs_ext):
+
+        #   node                                 node              node
+        #     |                reduce              |
+        # transpose1         =========>        transpose3     or
+        #     |
+        # transpose2
+        if op == 'transpose':
+            axes = eval(sym.attr('axes'))
             if childs[0].attr('op_name') == 'transpose':
                 caxes = eval(childs[0].attr('axes'))
                 axes = tuple([caxes[ii] for ii in axes])
-                consec = True
-            if consec:
                 sym = sym_iter(childs[0].get_children())[0]
                 if axes != tuple(sorted(axes)):
                     name = sym.attr('name') + "_fusetranspose"
                     sym = mx.sym.transpose(sym, axes=axes, name=name)
+
+        #   node                                 node
+        #     |                switch              |
+        # transpose1         =========>          relu
+        #     |                                    |
+        #   relu                               transpose1
+        elif op == 'relu':
+            if childs[0].attr('op_name') == 'transpose':
+                axes, name = eval(childs[0].attr('axes')), childs[0].attr('name')
+                sym = sym_iter(childs[0].get_children())[0]
+                sym = mx.sym.relu(sym)
+                sym = mx.sym.transpose(sym, axes=axes, name=name)
+
+        #     node     ...    node
+        #       |               |
+        #   transpose1 ...  transpose1                     node    node
+        #       \               /              reduce        \       /
+        #             concat                 =========>        concat
+        #                                                        |
+        #                                                    transpose1
+        elif op == 'Concat':
+            same, axeses = True, set()
+            for child in childs:
+                if child.attr('op_name') != 'transpose':
+                    same = False
+                    break
+                axeses.add(child.attr('axes'))
+            if same and len(axeses) == 1:
+                clist, dim = [], int(sym.attr('dim'))
+                axes = eval(list(axeses)[0])
+                for child in childs:
+                    cchild = sym_iter(child.get_children())[0]
+                    clist.append(cchild)
+                sym = mx.sym.concat(*clist, dim=axes[dim])
+                name = 'fusetranspose_' + name
+                sym = mx.sym.transpose(sym, axes=axes, name=name)
+
+        #    node                           node             
+        #      |              reduce          |
+        #  transpose1       =========>       sum
+        #      |
+        #     sum
+        # 
+        # only when 'keepdims' == False
+        # defalt: 'exclude' == False
+        elif op == 'sum':
+            axis, keepdims = eval(sym.attr('axis')), eval(sym.attr('keepdims'))
```

Comment:
It will throw an exception once the operator `sum` doesn't have the attribute `keepdims`.

```suggestion
            attr = sym.list_attr()
            axis, keepdims = get_attr(attr, 'axis', None), get_attr(attr, 'keepdims', False)
```

Reply:
already fixed